experimental/segmenter/src/lstm.rs

```diff
@@ -8,8 +8,9 @@ use std::char::decode_utf16;
 // TODO:
 // json file is big, So I should use anoher binary format like npy.
 // But provided npy uses tensorflow dtype.
-const THAI_MODEL: &[u8; 373466] =
-    include_bytes!("../data/Thai_codepoints_exclusive_model4_heavy/weights.json");
+const THAI_MODEL: &[u8; 373466] = include_bytes!(
+    "../../segmenter_lstm/tests/testdata/Thai_codepoints_exclusive_model4_heavy/weights.json"
```

Comment:
Note: it probably makes more sense to put the data elsewhere so that it can be used by `segmenter` and `segmenter_lstm`. In the future, we might also want to add Burmese training data from https://github.com/unicode-org/lstm_word_segmentation.

The purpose of the change is not to add the same Thai data again into icu4x because it is large.